server/gitlab/api.go

```diff
@@ -276,36 +290,59 @@ func (g *gitlab) GetReviews(ctx context.Context, user *UserInfo) ([]*MergeReques
 	opened := stateOpened
 	scope := scopeAll
 
-	var result []*internGitlab.MergeRequest
-
+	var mrs []*internGitlab.MergeRequest
 	if g.gitlabGroup == "" {
-		result, _, err = client.MergeRequests.ListMergeRequests(&internGitlab.ListMergeRequestsOptions{
-			ReviewerID: internGitlab.ReviewerID(user.GitlabUserID),
-			State:      &opened,
-			Scope:      &scope,
-		},
-			internGitlab.WithContext(ctx),
-		)
+		opt := &internGitlab.ListMergeRequestsOptions{
+			AssigneeID:  internGitlab.AssigneeID(user.GitlabUserID),
```

Comment:
Wondering why this changed from `ReviewerID` to `AssigneeID`

Reply:
Very good catch :+1: That's a bad merge. Submitted https://github.com/mattermost/mattermost-plugin-gitlab/pull/368